apps/whale-api/src/module.api/stats.controller.ts

```diff
@@ -96,6 +104,20 @@ export class StatsController {
     }
   }
 
+  @Get('/rewards/distribution')
```

Comment:
should add a test for this new endpoint in `stats.controller.spec.ts`?

Reply:
hmm.. the `spec` is for unit testing.. so yea i created another `e2e` for it  :grimacing:

Reply:
hmm okay actually that's fine then - e2e should cover it